### PR TITLE
fix: remove wrong message in abort_on_panic macro

### PR DESCRIPTION
**Pull request by themanforfree**
Please briefly answer these questions:

* what problem are you trying to solve? (or if there's no problem, what's the motivation for this change?)
remove wrong message in abort_on_panic macro
* what changes does this pull request make?
remove wrong message in abort_on_panic macro
* are there any non-obvious implications of these changes? (does it break compatibility with previous versions, etc)
no

### CODE REVIEW

File: curp/tests/it/read_state.rs [hunk 1]
```diff
@@ -23,6 +23,7 @@ async fn read_state() {
             TestCommandResult::default(),
         );
     });
+    sleep_millis(10).await;
```
Comment (Phoenix500526) [inline curp/tests/it/read_state.rs:26]: Why do you sleep here?

File: test-macros/src/lib.rs [hunk 2]
```diff
@@ -5,12 +5,11 @@ use syn::{parse_macro_input, ItemFn, Stmt};
 #[proc_macro_attribute]
 pub fn abort_on_panic(_attr: TokenStream, item: TokenStream) -> TokenStream {
     let mut input: syn::ItemFn = parse_macro_input!(item as ItemFn);
-    let test_case = &input.sig.ident.to_string();
 
     let panic_hook: Stmt = syn::parse_quote! {
         std::panic::set_hook(Box::new(|info| {
             let stacktrace = std::backtrace::Backtrace::force_capture();
-            println!("`{}` panic! \n@info:\n{}\n@stackTrace:\n{}", #test_case, info, stacktrace);
+            println!("test panic! \n@info:\n{}\n@stackTrace:\n{}", info, stacktrace);
```
Comment (Phoenix500526) [inline test-macros/src/lib.rs:12]: Can we parse the test case name from the first stack frame?

File: curp/tests/it/read_state.rs (hunk 1 above)
Comment (themanforfree) [reply curp/tests/it/read_state.rs:26]: In order to ensure that the `put_cmd` is proposed before `fetch_read_state`

File: test-macros/src/lib.rs (hunk 2 above)
Comment (themanforfree) [reply test-macros/src/lib.rs:12]: the test case name is irregularity in backtrace, we can't parse the test names unless we force all test names to conform to the specified format

**Review by bsbds — APPROVED** [review]

**Review by Phoenix500526 — APPROVED** [review]

**Pull request merged**